apps/impala/src/impala/conf.py

```diff
@@ -275,6 +279,13 @@ def get_use_sasl_default():
   default=False
 )
 
+USE_THRIFT_HTTP_JWT = Config(
```

Comment:
Slightly overkill to have a flag but fine to keep for now in case it needs to stay decoupled until v1